src/net/types.h

```diff
@@ -28,6 +28,30 @@ struct NetworkOptions {
     int port { 0 };
 };
 
+struct BaseStatus {
+    
+};
+
+// dummy structure for testing
+struct CreatureStatus : public BaseStatus {
```

Comment:
Can you put these structs into a separate header file? I now lean towards "types.h" files being containers for constants and enums.